src/components/ConnectWallet/index.tsx

```diff
@@ -13,8 +13,18 @@ interface WalletProps {
 export default function Wallet({ disabled, onClickConnectWallet }: WalletProps) {
   // Attempt to connect eagerly on mount
   useEffect(() => {
-    connections.forEach(([wallet, _]) => (wallet.connectEagerly ? wallet.connectEagerly() : wallet.activate()))
-  }, [])
+    connections.forEach(([wallet, _]) => {
```

Comment:
If these messages are logged for every wallet, they should include the name of the wallet that is attempting to `connectEagerly`/`activate`.